index.html

```diff
@@ -58,19 +58,21 @@
             </div>
           </div>
         <div class="intro">
-            <iframe 
-                class="hidden"
-                src="http://player.twitch.tv/?channel={TWITCH_USER_NAME}" 
-                height="100%" 
-                width="100%" 
-                frameborder="0" 
-                scrolling="no"
-                allowfullscreen="true"
-            ></iframe>
-          <div class="intro-text">
-            <h1>Twitch Stream offline.</h1>
-            <p>Stream offline. Come back later and watch me play.</p>
-          </div>
+            <script src= "http://player.twitch.tv/js/embed/v1.js"></script>
+<div id="{PLAYER_DIV_ID}"></div>
+<script type="text/javascript">
+    var options = {
+        width: 854,
+        height: 480,
+        channel: "{nl_kripp}", 
+        //video: "{VIDEO_ID}"       
+    };
+    var player = new Twitch.Player("{PLAYER_DIV_ID}", options);
+    player.setVolume(0.5);
+    player.addEventListener(Twitch.Player.PAUSE, () => { console.log('Player is paused!'); });
```

Comment:
Instead of this `PAUSE` event we can hook into the `OFFLINE` and `ONLINE` events to add / remove the `hidden` class on the `player-video div` 

Much like how the `main.js` is interacting  with the embed `iframe` now.